scripts/release/release-ckeditor5.js

```diff
@@ -0,0 +1,130 @@
+const validatePackageToRelease = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/validatepackagetorelease' );
+const { getChangesForVersion } = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/changelog' );
+
+const log = logger();
+const cke5Path = path.resolve( __dirname, '..', '..' );
+const packageJsonPath = path.resolve( cke5Path, 'package.json' );
+const templatePath = path.resolve( cke5Path, 'scripts', 'release', 'template' );
+const packageJsonTemplatePath = path.resolve( templatePath, 'package.json' );
+const packageJsonTemplateCopy = require( packageJsonTemplatePath );
+
+// That files will be copied from source to template directory and will be released too.
+const additionalFiles = [
+	'CHANGELOG.md',
+	'LICENSE.md',
+	'README.md'
+];
+
+cli.provideToken()
+	.then( token => {
+		const gitVersion = versionUtils.getLastTagFromGit();
+		const changelogVersion = versionUtils.getLastFromChangelog();
+
+		log.info( 'Comparing versions - saved in changelog and as a tag...' );
+
+		// If the last tag is equal to version saved in changelog, we don't have new version for release.
+		if ( gitVersion === changelogVersion ) {
+			return reject( 'Before starting the release process, you should generate the changelog.' );
+		}
+
+		const releaseDescription = getChangesForVersion( changelogVersion );
+
+		log.info( 'Validating the repository for the release...' );
+
+		const errors = validatePackageToRelease( {
+			version: changelogVersion,
+			changes: releaseDescription
+		} );
+
+		// Abort due to errors during validation.
+		if ( errors.length ) {
+			const log = logger();
+
+			log.error( 'Unexpected errors occured:' );
+			errors.map( err => '* ' + err ).forEach( log.error.bind( log ) );
+
+			return reject( 'Releasing has been aborted due to errors.' );
+		}
+
+		log.info( 'Bumping the version...' );
+
+		// Bump the version.
+		tools.shExec( `npm version ${ changelogVersion } --message "Release: v${ changelogVersion }."`, { verbosity: 'error' } );
+
+		const packageJson = require( packageJsonPath );
+
+		log.info( 'Copying the package.json...' );
+
+		// Update the template of `package.json`. We will use values from source `package.json`
+		// but only these ones which are defined in the template.
+		tools.updateJSONFile( packageJsonTemplatePath, jsonFile => {
+			for ( const property of Object.keys( jsonFile ) ) {
+				// If the `property` is set in the template, leave it.
+				if ( jsonFile[ property ] ) {
+					continue;
+				}
+
+				// Copy value from original package.json file.
+				jsonFile[ property ] = packageJson[ property ];
+			}
+
+			return jsonFile;
+		} );
+
+		// Copy additional files.
+		for ( const file of additionalFiles ) {
+			tools.shExec( `cp ${ path.resolve( cke5Path, file ) } ${ path.resolve( templatePath, file ) }` );
+		}
+
+		log.info( 'Publishing on npm...' );
+
+		// Publish the package on npm.
+		tools.shExec( `cd ${ templatePath } && npm publish && cd ${ cke5Path }` );
```

Comment:
If you would like to test it – replace `npm publish` with `npm pack`. Remember about adding `exit 1;` to the shell script (`scripts/release-ckeditor5.sh`) in L15 and committing these changes because NPM throws an error if the repository is dirty.

Reply:
After finishing the test phase, remember about removing the tag created by `npm version`. :D